packages/component/src/Activity/Bubble.js

```diff
@@ -23,14 +23,14 @@ const Bubble = ({
   </div>;
 
 Bubble.defaultProps = {
-  'aria-label': '',
+  'aria-hidden': true,
```

Comment:
It seems weird to have it by default to `true`. People could unknowingly hit a bug.

```suggestion
  'aria-hidden': false,
```